documentation/src/main/asciidoc/topics/ref_eviction_examples.adoc

```diff
@@ -1,92 +1,92 @@
 
-Passivation persists data to cache stores when {brandname} evicts entries. You
-should always enable eviction if you enable passivation.
+{brandname} stores cache entries as objects in the JVM heap. Eviction happens
+when there are 100 entries in the data container and {brandname} gets a request
+to create a new entry:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<persistence passivation="true">
-  ...
-</persistence>
-
-<memory />
+<memory max-count="100"/>
 ----
 
-.Manual eviction
+.Eviction based maximum size in bytes
+
+{brandname} can store cache entries as `byte[]` arrays if you encode entries in binary storage formats, for example: `application/x-protostream` format.
 
-{brandname} stores cache entries as objects. Eviction is not enabled but
-performed manually using the `evict()` method.
+In the following example, {brandname} performs eviction when the size of the
+data container reaches 500 MB (megabytes) in size and it gets a request to
+create a new entry:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<memory>
-  <object strategy="MANUAL" />
-</memory>
+<encoding media-type="application/x-protostream"/> <1>
+<memory max-size="500 MB"/> <2>
 ----
 
-.Object storage with eviction
+<1> Specifies a binary format for entries in the cache.
+<2> Defines the maximum size of the data container as MB (megabytes).
 
-{brandname} stores cache entries as objects. Eviction happens when there are
-100 entries in the data container and {brandname} gets a request to create a
-new entry:
+.Off-heap storage
+
+{brandname} stores cache entries as bytes in native memory. Eviction happens
+when there are 100 entries in the data container and {brandname} gets a request
+to create a new entry:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<memory>
-  <object size="100" />
-</memory>
+<memory storage="OFF_HEAP" max-count="100"/>
 ----
 
-.Binary storage with memory-based eviction
+.Off-heap storage with the exception strategy
 
-{brandname} stores cache entries as bytes. Eviction happens when the size of
-the data container reaches 100 bytes and {brandname} gets a request to create a
-new entry:
+{brandname} stores cache entries as bytes in native memory. When there are 100
+entries in the data container, and {brandname} gets a request to create a new
+entry, it throws an exception and does not allow the new entry:
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-<memory>
-  <binary size="100" eviction="MEMORY"/>
-</memory>
+<memory storage="OFF_HEAP" max-count="100" when-full="EXCEPTION"/>
 ----
 
-.Off-heap storage with count-based eviction
+.Manual eviction
 
-{brandname} stores cache entries as bytes in native memory. Eviction happens
-when there are 100 entries in the data container and {brandname} gets a request
-to create a new entry:
+{brandname} stores cache entries as objects in the JVM heap. Eviction is not
+enabled but performed manually using the `evict()` method.
```

Comment:
Should we mention this does nothing but prevents a warning message when passivation is enabled but there is no eviction configuration?